docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -156,17 +157,56 @@ dotnet_naming_symbols.public_symbols.applicable_kinds           = property,metho
 dotnet_naming_symbols.public_symbols.applicable_accessibilities = public
 dotnet_naming_symbols.public_symbols.required_modifiers         = readonly
 
-# Defining the `first_word_upper_case_style` naming style
+# Defining the 'first_word_upper_case_style' naming style
 dotnet_naming_style.first_word_upper_case_style.capitalization = first_word_upper
 
-# Defining the `public_members_must_be_capitalized` naming rule, by setting the symbol group to the 'public symbols' symbol group,
-dotnet_naming_rule.public_members_must_be_capitalized.symbols   = public_symbols
-# setting the naming style to the `first_word_upper_case_style` naming style,
+# Defining the 'public_members_must_be_capitalized' naming rule, by setting the
+# symbol group to the 'public symbols' symbol group,
+dotnet_naming_rule.public_members_must_be_capitalized.symbols  = public_symbols
+# setting the naming style to the 'first_word_upper_case_style' naming style,
 dotnet_naming_rule.public_members_must_be_capitalized.style    = first_word_upper_case_style
 # and setting the severity.
 dotnet_naming_rule.public_members_must_be_capitalized.severity = suggestion
 ```
 
+## Example: Private instance fields with underscore
+
+This `.editorconfig` file snippet enforces that private instance fields should start with an `_`; if that convention is not followed, the IDE will treat it as a compiler error. Private static fields will be ignored.
+
+Because you can only define a symbol group based on the identifiers it has (e.g. `static` or `readonly`), and not by the identifiers it doesn't have (e.g. an instance field because it doesn't have `static`), you need to define two naming rules:
```

Comment:
```suggestion
Because you can only define a symbol group based on the identifiers it has (for example, `static` or `readonly`), and not by the identifiers it doesn't have (for example, an instance field because it doesn't have `static`), you need to define two naming rules:
```